flink-connector/src/main/java/org/apache/flink/connector/delta/sink/writer/DeltaWriterBucketState.java

```diff
@@ -47,10 +47,18 @@
 
```

Comment:
update docs saying that `extends the state by adding application's unique identifier and checkpoint interval identifier to guarantee....`